contrib/builder/rpm/armhf/centos-7/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM multiarch/centos:7.2.1511-armhfp-clean
+
+RUN yum groupinstall --skip-broken -y "Development Tools"
+RUN yum -y swap -- remove systemd-container systemd-container-libs -- install systemd systemd-libs
+RUN yum install -y btrfs-progs-devel device-mapper-devel glibc-static libseccomp-devel libselinux-devel libtool-ltdl-devel pkgconfig selinux-policy selinux-policy-devel sqlite-devel systemd-devel tar git cmake vim-common
+
+ENV GO_VERSION 1.7.3
```

Comment:
can you bump this to 1.7.4? I'm setting up my arm box now to test this, hopefully we can get this in before it has to be bumped again to 1.8